packages/gatsby-plugin-manifest/README.md

```diff
@@ -188,9 +188,11 @@ module.exports = {
 }
 ```
 
-## Custom icon options
+## Iterative icon options
```

Comment:
Calling this is an iterator is kinda weird, no?

I think I'd just re-word this slightly

```suggestion
## Custom icon options
```